vital/algo/estimate_similarity_transform.h

```diff
@@ -47,10 +55,13 @@ class VITAL_ALGO_EXPORT estimate_similarity_transform
    *          \c from space to points in the \c to space.
    */
   virtual kwiver::vital::similarity_d
-  estimate_transform(std::vector<kwiver::vital::vector_3d> const& from,
-                     std::vector<kwiver::vital::vector_3d> const& to) const = 0;
+  estimate_transform( std::vector< kwiver::vital::vector_3d > const& from,
+                      std::vector< kwiver::vital::vector_3d > const& to ) const
+  = 0;
```

Comment:
should the const be on this line?

Reply:
I agree that the ``const`` should be on the same line as the ``= 0;``.   However, if the ``= 0;`` is moved up with the ``const`` then the line is too long.  However,  I think moving ``const = 0;`` to the next line leaves the code less clear.  It's my opinion that this is a special case where exceeding the line length may be acceptable for clarity & readability.  I recognize that others' opinions may be different.  I'll change it to my preference and we'll see if anyone complains.